templates_jinja2/team_admin_redeem.html

```diff
@@ -35,7 +35,9 @@ <h1>Redeem Team Admin Code</h1>
   </ul>
   <a class="btn btn-primary" href="/mod"><span class="glyphicon glyphicon-eye-open"></span> View Mod Dashboard</a>
 {% else %}
-  <p>Redeem an access code to moderate a team's media. More help <a href="/mod/help">can be found here</a>.</p>
+  <p>Redeem an access code to moderate a team's media. Having moderator access allows you to manage media linked to the team's TBA profile as well as accept/reject suggestions for additional media.</p>
```

Comment:
What about "...moderate your team's media" instead of "a"?

Reply:
:+1: